python/src/scipp/plot/plot_3d.py

```diff
@@ -329,18 +337,22 @@ def get_meshes(self):
         return meshes
 
     def get_box(self):
-        max_size = 0.0
-        dx = {"x": 0, "y": 0, "z": 0}
-        for ax in dx.keys():
-            dx[ax] = (self.xminmax[self.button_axis_to_dim[ax]][1] -
-                      self.xminmax[self.button_axis_to_dim[ax]][0])
-        max_size = np.amax(list(dx.values()))
-        arrays = dict()
-        for ax, size in dx.items():
-            diff = max_size - size
-            arrays[ax] = [
-                self.xminmax[self.button_axis_to_dim[ax]][0] - 0.5*diff,
-                self.xminmax[self.button_axis_to_dim[ax]][1] + 0.5*diff]
-
-        return np.meshgrid(arrays["x"], arrays["y"], arrays["z"],
+        # max_size = 0.0
+        # dx = {"x": 0, "y": 0, "z": 0}
+        # for ax in dx.keys():
+        #     dx[ax] = (self.xminmax[self.button_axis_to_dim[ax]][1] -
+        #               self.xminmax[self.button_axis_to_dim[ax]][0])
+        # max_size = np.amax(list(dx.values()))
+        # arrays = dict()
+        # for ax, size in dx.items():
+        #     diff = max_size - size
+        #     arrays[ax] = [
+        #         self.xminmax[self.button_axis_to_dim[ax]][0] - 0.5*diff,
+        #         self.xminmax[self.button_axis_to_dim[ax]][1] + 0.5*diff]
+
+        # # return np.meshgrid(arrays["x"], arrays["y"], arrays["z"],
+        # #                    indexing="ij")
```

Comment:
Commented code?

Reply:
ah yes, this was for conserving aspect ratio. I will make it into a keyword argument, like i did for `plot_2d`